docs/metrics/extending-the-sdk/README.md

```diff
@@ -27,8 +27,11 @@ not covered by the built-in exporters:
   does not implement any retry logic.
 * Exporters should avoid generating telemetry and causing live-loop, this can be
   done via `OpenTelemetry.SuppressInstrumentationScope`.
-* Exporters receives a batch of `Metric`, and each `Metric`
-  can contain 1 or more `MetricPoint`s.
+* Exporters receives a batch of `Metric`, and each `Metric` can contain 1 or
+  more `MetricPoint`s. The exporter should perform all actions with the
+  `Metric`s and `MetricsPoint`s in the Batch before returning control from
+  `Export`, once the control is returned, the exporter can no longer assume the
```

Comment:
nit: Rephrase to "exporter can no longer make any assumptions about the state of the Batch..."